docs/administration/metrics/_index.md

```diff
@@ -59,6 +63,21 @@ Name | Description | Labels (Values) |Metric type
 `registry_http_request_size_bytes` | The HTTP request sizes in bytes. | handler, le | histogram
 {{< /table >}}
 
+## Harbor Jobservice metrics
+
+The following are metrics pulled from the Docker distribution and are available at `<harbor_instance>:<metrics_port>/<metrics_path>?comp=jobservice`.
```

Comment:
these metrics are pulled from the harbor's jobservice component, not Docker distribution

Reply:
thanks @ninjadq! this is updated.